tests/test_cli.py

```
@@ -3286,5 +3308,166 @@ def mock_exists_side_effect(path):
+    """Test Kubernetes command handling."""
+
+    def setUp(self):
+        """Set up test fixtures."""
+        self.mock_args = MagicMock()
+
+    @patch("builtins.input")
+    @patch("builtins.print")
+    @patch("mmrelay.k8s_utils.generate_manifests")
+    @patch("mmrelay.k8s_utils.prompt_for_config")
+    def test_handle_k8s_command_generate_manifests_success(
+        self, mock_prompt, mock_generate, mock_print, mock_input
```

Ruff (0.14.13)

Unused method argument: `mock_print`

(ARG002)